app/deck/1.8.x/guides/defaults.md

```diff
@@ -0,0 +1,380 @@
+
+{:.note}
+> **Note:** The following are only properties that **have defaults**, and are
+not all of the available properties for each object.
+
+{% navtabs %}
+{% navtab Route %}
+
+Set the following properties to the values you want to use across all Routes:
+
+```yaml
+_info:
+  defaults:
+    route:
+      https_redirect_status_code: 426
+      path_handling: v0
+      preserve_host: false
+      protocols:
+      - http
+      - https
+      regex_priority: 0
+      request_buffering: true
+      response_buffering: true
+      strip_path: true
+```
+
+For all available properties, see the
+[Route object](/gateway-oss/latest/admin-api/#route-object) documentation.
+
+{% endnavtab %}
+{% navtab Service %}
+
+Set the following properties to the values you want to use across all Services:
+
+```yaml
+_info:
+  defaults:
+    service:
+      port: 80
+      protocol: http
+      connect_timeout: 60000
+      write_timeout: 60000
+      read_timeout: 60000
+      retries: 5
+```
+For all available properties, see the
+[Service object](/gateway-oss/latest/admin-api/#service-object) documentation.
+
+{% endnavtab %}
+{% navtab Upstream %}
+
+Set the following properties to the values you want to use across all Upstreams:
+
+```yaml
+_info:
+  defaults:
+    upstream:
+      slots: 10000
+      algorithm: round-robin
+      hash_fallback: none
+      hash_on: none
+      hash_on_cookie_path: /
+      healthchecks:
+        active:
+          concurrency: 10
+          healthy:
+            http_statuses:
+            - 200
+            - 302
+            interval: 0
+            successes: 0
+          http_path: /
+          https_verify_certificate: true
+          timeout: 1
+          type: http
+          unhealthy:
+            http_failures: 0
+            http_statuses:
+            - 429
+            - 404
+            - 500
+            - 501
+            - 502
+            - 503
+            - 504
+            - 505
+            interval: 0
+            tcp_failures: 0
+            timeouts: 0
+        passive:
+          healthy:
+            http_statuses:
+            - 200
+            - 201
+            - 202
+            - 203
+            - 204
+            - 205
+            - 206
+            - 207
+            - 208
+            - 226
+            - 300
+            - 301
+            - 302
+            - 303
+            - 304
+            - 305
+            - 306
+            - 307
+            - 308
+            successes: 0
+          type: http
+          unhealthy:
+            http_failures: 0
+            http_statuses:
+              - 429
+              - 500
+              - 503
+            tcp_failures: 0
+            timeouts: 0
+        threshold: 0
+```
+For all available properties, see the
+[Upstream object](/gateway-oss/latest/admin-api/#upstream-object) documentation.
+
+{% endnavtab %}
+{% navtab Target %}
+Targets have no required default properties. However, you can set custom
+defaults as desired.
```

Comment:
Not sure what you mean by "custom".
`Weight` is what we default I think (please recheck)

Reply:
Thanks for the pointer to the /schemas endpoint (on the ticket), I didn't know that existed. Checked it,`weight` defaults to 100. I'll adjust.